storm-core/src/jvm/backtype/storm/drpc/DRPCSpout.java

```diff
@@ -129,7 +133,9 @@ private void checkFutures() {
     public void open(Map conf, TopologyContext context, SpoutOutputCollector collector) {
         _collector = collector;
         if(_local_drpc_id==null) {
-            _backround = Executors.newCachedThreadPool();
+            _backround = new ExtendedThreadPoolExecutor(0, Integer.MAX_VALUE,
+                60L, TimeUnit.SECONDS,
+                new SynchronousQueue<Runnable>());
```

Comment:
@lazyval the `newCachedThreadPool` uses these very arguments to generate `ThreadPoolExcecutor` for cached threads. So it won't have any additional performance overhead.